package proto_ff;

import "yd_fieldoptions.proto";


message E_TaskrewardTaskrewardAttrDesc
{
	optional int64 m_val = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int64 m_id = 2[(yd_fieldoptions.field_cname) = "id"];
}

message E_TaskrewardTaskrewardItemDesc
{
	optional int32 m_box = 1[(yd_fieldoptions.field_cname) = "掉落"];
	optional int32 m_prof = 2[(yd_fieldoptions.field_cname) = "职业"];
	optional int32 m_bind = 3[(yd_fieldoptions.field_cname) = "绑定"];
	optional int32 m_val = 4[(yd_fieldoptions.field_cname) = "数量"];
	optional int64 m_id = 5[(yd_fieldoptions.field_cname) = "id"];
}

message E_TaskrewardTaskreward
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "奖励id"];
	optional int32 m_taskType = 2[(yd_fieldoptions.field_cname) = "任务类型"];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 m_guildExp = 4[(yd_fieldoptions.field_cname) = "公会经验"];
	optional int32 m_guildPoint = 5[(yd_fieldoptions.field_cname) = "公会贡献"];
	optional int32 m_skillId = 6[(yd_fieldoptions.field_cname) = "技能ID"];
	repeated E_TaskrewardTaskrewardAttrDesc m_attr = 7[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 5];
	repeated E_TaskrewardTaskrewardItemDesc m_item = 8[(yd_fieldoptions.field_cname) = "物品", (yd_fieldoptions.field_arysize) = 17];
}


message Sheet_TaskrewardTaskreward
{
	repeated E_TaskrewardTaskreward E_TaskrewardTaskreward_List = 1[(yd_fieldoptions.field_arysize)=4096];
}

message E_TaskrewardTasktype
{
	optional int32 m_taskType = 1[(yd_fieldoptions.field_cname) = "任务类型"];
}


message Sheet_TaskrewardTasktype
{
	repeated E_TaskrewardTasktype E_TaskrewardTasktype_List = 1[(yd_fieldoptions.field_arysize)=4];
}
